[CAM-11123] Cloud Academy: review documentation explaining CAM troubleshooting login issues (#2752)
CenturyLinkCloud/PublicKB: Cloud Application Manager/Getting Started/getting-started-create-user.md

![Getting Started - Login page](../../images/cloud-application-manager/getting-started-login-1.png)

> For more information, you see [authentication options](../../Administering Your Organization/user-authentication.md)

Reviewer: ```suggestion
For more information, you can see [authentication options](../../Administering Your Organization/user-authentication.md)
```

> #### Troubleshoot login mechanisms

Reviewer: Please, use three level title and include in ToC

Reviewer: Please, address this comment as well

> When the credentials are authentication by CAM and there is any problem, this problem is shown and it indicates which is the problem.

Reviewer: ```suggestion
When the credentials are authenticated by CAM and there is any issue, an error message is shown to indicate the cause of the issue.
```

> ![Getting Started - Login page](../../images/cloud-application-manager/management/cam-error.png)

Reviewer: Change alt text

Reviewer: Please, address this comment as well

> But if credentials are authentication by other servers different at CAM. The server is who gives you the information. For example if en SAML the user or/and incorrect, SAML server shown:

Reviewer: ```suggestion
If you are using an authentication method other than username and password (this is the only one internal to Cloud Application Manager), the issue will probably come from the authentication system being used, so the authentication server used is the one returning the error. For example if you are using SAML with an incorrect username and/or password, the SAML server returns:
```

> ![Getting Started - Login page](../../images/cloud-application-manager/management/saml-error.png)

Reviewer: Change alt text

Reviewer: Please, address this comment as well

> If the server is broken, then you will have a 404 Error.

Reviewer: ```suggestion
If the server is down or having an issue, then the message will reflect that, but this does not mean that Cloud Application Manager is down. In that case, if you have any other authentication method set, you can try it to workaround the issue with your first authentication method choice, and still be able to log in into Cloud Application Manager and work normally.
```

![Getting Started - Login page](../../images/cloud-application-manager/getting-started-login-1.png)

For more information, you can see [authentication options of your Organization](../../Administering Your Organization/user-authentication.md)

> ### Troubleshoot login mechanisms

Reviewer: ```suggestion
### Troubleshooting login mechanisms
```

When the credentials are authenticated by CAM and there is any issue, an error message is shown to indicate the cause of the issue.

> ![Example of CAM error](../../images/cloud-application-manager/management/cam-error.png)

Reviewer: ```suggestion
![Example of CAM login error message](../../images/cloud-application-manager/management/cam-error.png)
```

If you are using an authentication method other than username and password (this is the only one internal to Cloud Application Manager), the issue will probably come from the authentication system being used, so the authentication server used is the one returning the error. For example if you are using SAML with an incorrect username and/or password, the SAML server returns:

> ![Example SAML error](../../images/cloud-application-manager/management/saml-error.png)

Reviewer: ```suggestion
![Example SAML login error message](../../images/cloud-application-manager/management/saml-error.png)
```
